src/app/console.py

```diff
@@ -1,58 +1,188 @@
-from src.domain.baraba import Baraba
-from src.domain.hero import Hero
-from src.domain.zemlja import Zemlja
+from dataclasses import dataclass
+
 import random
+import sys
 
 
+@dataclass
 class Console():
     def __init__(self):
+
         self.zemlja = None
         self.level = 1
+        self.maxKoraki = 5
+        self.rezultat = 0
 
     def new_game(self):
         barabe: list[Baraba] = []
         stBarab = self.level * 2
+        sirina = 13 - self.level
+        visina = 13 - self.level
+        self.maxKoraki = 5
+        # generiraj vse mozne kombinacije matrice
+        moznePozicije = [(x, y) for x in range(sirina) for y in range(visina)]
 
         for i in range(stBarab):
+            # izberi med pozicijami, ki so še na voljo
+            x, y = random.choice(moznePozicije)
             barabe.append(
                 Baraba(
-                    x=random.randint(0, 10 * self.level),
-                    y=random.randint(0, 10 * self.level),
+                    x=x,
+                    y=y,
                     hitrost=2)
             )
+            # vsakic odstrani pozicijo barabe iz moznih pozicij
+            moznePozicije.remove((x, y))
+        # izberi pozicije za heroja in princesko
+        xh, yh = random.choice(moznePozicije)
+        moznePozicije.remove((xh, yh))
+        xp, yp = random.choice(moznePozicije)
+        moznePozicije.remove((xp, yp))
 
         self.zemlja = Zemlja(
-            sirina=10 * self.level,
-            dolzina=10 * self.level,
+            sirina=sirina,
+            visina=visina,
             hero=Hero(
-                x=random.randint(0, 10 * self.level),
-                y=random.randint(0, 10 * self.level)),
+                x=xh,
+                y=yh),
+            princeska=Princeska(
+                x=xp,
+                y=yp),
             barabe=barabe
         )
 
     def draw_game(self):
-        for y in range(self.zemlja.dolzina+1):
-            for x in range(self.zemlja.sirina+1):
+        print('draw game')
+        print(f'\033[1m\033[33m{self.level} LEVEL\033[0m')
```

Comment:
Ce hoces imeti barve moras poskrbeti da bo cross platform ker linux in windows nimata istih kod za barve zato raje uporabljaj knjižnice ki podpirajo to... https://pypi.org/project/colored/